build.gradle

```diff
@@ -0,0 +1,83 @@
+apply plugin: "idea"
+apply plugin: "java"
+apply plugin: "groovy"
+version = "0.0.2"
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+ext {
+    sourceEncoding = "UTF-8"
+//testing
+    spockVersion = "1.0-groovy-2.3"
+    groovyVersion = "2.3.3"
+    junitVersion = "4.12"
+    cglibVersion = "3.2.0"
+    objenesisVersion = "2.2"
+    jaxbVersion = "2.2.12"
+    betamaxVersion = "2.0-SNAPSHOT"
+
+    libs = [
+            jaxb         : "javax.xml.bind:jaxb-api:$jaxbVersion",
+            junit        : "junit:junit:$junitVersion",
+            groovy       : "org.codehaus.groovy:groovy-all:$groovyVersion",
+            spock        : "org.spockframework:spock-core:$spockVersion",
+            cglib        : "cglib:cglib-nodep:$cglibVersion",
+            objenesis    : "org.objenesis:objenesis:$objenesisVersion",
+            betamax_proxy: "co.freeside.betamax:betamax-proxy:$betamaxVersion",
+            betamax_junit: "co.freeside.betamax:betamax-junit:$betamaxVersion"
+    ]
+
+    javadocFolder = "doc"
+
+    sharedManifest = manifest {
+        attributes("Implementation-Title": 'Twingly Search API Java',
+                "Implementation-Version": version)
+    }
+}
+
+repositories {
+    mavenCentral()
+    jcenter()
+    mavenLocal()
+    maven { url "http://oss.jfrog.org/oss-snapshot-local/" }
+}
+dependencies {
+    compile libs.jaxb
+    testCompile libs.junit
+    testCompile libs.groovy
+    testCompile libs.spock
+    testRuntime libs.cglib
+    testRuntime libs.objenesis
+    testCompile libs.betamax_proxy
+    testCompile libs.betamax_junit
+}
+
+test {
+    testLogging.showStandardStreams = true
+    testLogging {
+        events "PASSED", "STARTED", "FAILED", "SKIPPED"
```

Comment:
Might be that I am more used to just seeing one line per test. We can always add `STARTED` back later on, if we change our minds.